cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -124,6 +124,74 @@
       player.controls = {{ (not embed_config.controlsOff) | default(True) | tojson }};
       player.muted = {{ embed_config.muted | default(False) | tojson }};
 
+      // Add chapters if they exist
+      {% if record and record.get("metadata", {}).get("description") %}
+      function parseChapters(description) {
+        if (!description) return [];
+        
+        const pattern = /(?:^|\n)\s*(\d{1,2}:(?:\d{1,2}:)?\d{2})\s*[-\s]*(.+?)(?=\n|$)/gm;
```

Comment:
Minor: On the python check in the utils.py we allow single digits as well as seconds, here we should as well to be consistent.

```suggestion
        const pattern = /(?:^|\n)\s*(\d{1,2}:(?:\d{1,2}:)?\d{1,2})\s*[-\s]*(.+?)(?=\n|$)/gm;
```